blocks/library/heading/test/index.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import { query, getBlocks, unregisterBlock, setUnknownTypeHandler } from 'api';
+import { blockSettings } from '../';
+
+describe( 'heading block', () => {
+	afterEach( () => {
```

Comment:
>  a more immediate solution might be to move the registerBlock call out of library and simply have the block export its settings. This doesn't set as good an example for third-party block implementers though, for which this pattern may or may not be the best.

That was my first intention, but I decided not to do it because we'll need to export the "slug" as a separate variable too. Maybe we should include the slug in the exported object? What do you think? And like you said, This doesn't set as good an example for third-party block implementers.